src/library.js

```diff
@@ -2792,7 +2812,7 @@ LibraryManager.library = {
         return leadingNulls(Math.ceil(daysDifference/7), 2);
       },
       '%w': function(date) {
-        var day = new Date(date.tm_year+1900, date.tm_mon+1, date.tm_mday, 0, 0, 0, 0);
+        var day = new Date(date.tm_year+1900, date.tm_mon, date.tm_mday, 0, 0, 0, 0);
```

Comment:
The fact that this didn't change any test results makes me think we need more test coverage here.

Reply:
I don't think tests actually ran here. @abujalski, this and your other PRs all seem to not have CI run on the commits at all. Any idea why? It's very strange...

Reply:
Indeed, according to outputs there's something wrong with .circleci/config.yml. I've run locally latest config validator https://circleci.com/docs/2.0/local-cli/#validate-a-circleci-config and it showed that there is duplicate 'steps' section in test-browser-firefox.

Problematic lines where added in ca9520c87c long time ago, so maybe there was some update of CircleCI validator and now it is more strict. Question is why this happened only for my fork.

Commit https://github.com/abujalski/emscripten/commit/743f25f618157b0228af1a240ad97846dde9b043 removes one of them and now CI runs.

However, now the test-upstream-browser-chrome are failing. I'll update PR when I check why.

Reply:
@sbc100 added more tests in https://github.com/emscripten-core/emscripten/pull/8630/commits/ff33f26ed2708b215a571aebe0859c979ad8a923 to check '%w' and '%u' format specifier.

BTW. I found that according to the spec https://pubs.opengroup.org/onlinepubs/9699919799/functions/strftime.html (and gcc on Ubuntu PC) this two formats should take only `tm_wday` field.